test/test_rpc.py

```diff
@@ -90,7 +92,16 @@ def wrapper(self):
     sys.version_info < (3, 0),
     "Pytorch distributed rpc package " "does not support python2",
 )
+@unittest.skipIf(C.is_asan, "Skip ASAN since torch + multiprocessing + fbcode doesn't work with asan")
 class RpcTest(MultiProcessTestCase):
+    @classmethod
+    def setUpClass(cls):
+        multiprocessing.set_start_method("spawn")
+        for attr in dir(cls):
```

Comment:
Can you call `super(RpcTest, self). setUpClass()` instead, to avoid duplicating code?

Reply:
will remove sys.exit(0) after c10d use spawn, right now c10d is still using fork